src/azul/plugins/anvil.py

```diff
@@ -0,0 +1,104 @@
+from abc import (
```

Comment:
Can this live in `azul.plugins.metadata.anvil`?

Reply:
No (it causes a circular import), but an alternative was agreed upon at PL.